Makefile

```diff
@@ -289,7 +289,7 @@ test-client:
 test-object-clients:
 	# The parallelism is lowered here because these tests run several pachd
 	# deployments in kubernetes which may contest resources.
-	go test -count=1 ./src/internal/obj/testing -timeout $(TIMEOUT) -parallel=2
+	go test -count=1 ./src/internal/obj/obj_intgtest -timeout $(TIMEOUT) -parallel=2
```

Comment:
Out of curiosity - why was this renamed?   I have no clue what `intgtest` means - some googling got me to 'integration test'?  Can we not repeat `obj` in the path, at least?  Also, since the test code is now split between `obj` and `obj_intgtest`, this should probably run both or else we might miss some stuff in CI.

Reply:
I don't really care about the name, other than `testing` being a bad name because of the conflict with the standard library.  I thought it might be useful to distinguish these kinds of tests as integration tests (expensive to run, require a lot of config, use external services).  Maybe you would call them end-to-end tests.  Let me know if you have a strong preference for a particular name; I will change it.

I will also run the obj package tests.  The idea is that it is easier to iterate on the tests when they are in the obj package and running against the local client.  So that is where the test suite should live.  And then we call into the test suite + additional tests in the other package.

Reply:
I'd be fine with `integration_tests`.

I use `testing` packages to get around circular dependencies, usually, so they often don't need to be included (and the name conflict with the `testing` stdlib shouldn't matter).  In the case where we have behavior that other tests might want, it might be worth putting in a `testutil` package.

Reply:
There is an idiom in go: `_test` packages that handle the circular dependency case.

https://tip.golang.org/cmd/go/#hdr-Test_packages
>  Test files that declare a package with the suffix "_test" will be compiled as a separate package, and then linked and run with the main test binary. 

That is the only time you are allowed to have two packages in the same directory.

So in the pfs server, I would be in favor of renaming the testing package to pfs_test and just have those tests in the server package.  But here I think we should keep them in a separate package because they are more costly to run, and are a different kind of test. We aren't just getting around the circular dependency.

I will change to `integration_tests`

Reply:
I think we should avoid the underscore in the package name like @Tryneus suggested.